driverapi/driverapi.go

```diff
@@ -106,3 +103,76 @@ type DriverCallback interface {
 type Capability struct {
 	DataScope datastore.DataScope
 }
+
+// IPData represents the per-network ip related
+// operational information libnetwork will send
+// to the network driver during  CreateNetwork()
+type IPData struct {
```

Comment:
I can.

Sorry for the misunderstanding, I had removed it because in this [comment](https://github.com/docker/libnetwork/issues/489#issuecomment-140245594) you were saying you do not really need it.

I will add it back then.